install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -300,7 +300,9 @@ data:
   #   - disabled
   #   - vxlan (default)
   #   - geneve
+{{- if not .Values.gke.enabled }}
```

Comment:
Could you please move this check above the comment for the `tunnel` option, and add a comment line to mention that in the case of GKE this is set at another location in the file?

Reply:
Done!